Tests/StoreKitUnitTests/BasePurchasesOrchestratorTests.swift

```diff
@@ -0,0 +1,254 @@
+//
+//  Copyright RevenueCat Inc. All Rights Reserved.
+//
+//  Licensed under the MIT License (the "License");
+//  you may not use this file except in compliance with the License.
+//  You may obtain a copy of the License at
+//
+//      https://opensource.org/licenses/MIT
+//
+//  PurchasesOrchestratorTests.swift
+//
+//  Created by Andrés Boedo on 1/9/21.
+
+import Foundation
+import Nimble
+@testable import RevenueCat
+import StoreKit
+import XCTest
+
+@available(iOS 14.0, tvOS 14.0, macOS 11.0, watchOS 7.0, *)
+class BasePurchasesOrchestratorTests: StoreKitConfigTestCase {
```

Comment:
This file appears as new but it's really just a renamed `PurchasesOrchestratorTests.swift` => `BasePurchasesOrchestratorTests.swift`